### ISSUE TITLE
Les events invalides (partiellement) faussent les résultats .csv. (ex de Bagnols et Nans les pins 83)

### ISSUE THREAD

**Issue by CeliaVermicelli** [MTES-MCT/Docurba#761]
[BAGNOLS](https://docurba.beta.gouv.fr/ddt/83/collectivites/83008/commune) (83) : l'approbation du PLU du 05/04/2013 n'apparait pas dans les extractions car notée en "invalide" alors qu'elle n'est que partiellement annulée,

[NANS LES PINS](https://docurba.beta.gouv.fr/ddt/83/collectivites/83087/commune) : l'approbation du PLU du 13/10/2020 n'apparait pas dans les extractions car notée en "invalide" alors qu'elle n'est que partiellement annulée.

En lien avec https://github.com/MTES-MCT/Docurba/issues/344 -> un event invalide n'est pas non plus dans les events clefs, alors que invalidés **partiellement** sur Sudocuh.

**Comment by HermanceGauth** [comment]
@UngererFabien @lerayj en lien avec le premier point remonté par Claire dans son mail du 23 août, annulation partielle doit rester valide et c'est ce qu'il y a a priori dans le code, étrange que vous n'ayez pas vu ce point lors des comparaisons ligne par ligne, peut-être quelque chose qui a changé depuis?
![image](https://github.com/user-attachments/assets/bd615e61-fd24-4d07-a12f-71dd47671d68)

**Comment by HermanceGauth** [comment]
@CeliaVermicelli corrigé ce jour peut-être avec la #1115 ? Pose la question des event invalid mais je fais une issue à  part sur le sujet.

**Comment by CeliaVermicelli** [comment]
yes c'est corrigé pour les deux cas du 83 - je close cette issue puisque tu as créée https://github.com/MTES-MCT/Docurba/issues/1180
